core/src/main/java/org/mule/runtime/core/internal/construct/AbstractFlowConstruct.java

```diff
@@ -113,9 +120,12 @@ public final void initialise() throws InitialisationException {
 
   @Override
   public final void start() throws MuleException {
+    boolean ignoreInitialState =
```

Comment:
make declarative this local variable. This doesn't simply ignore the initial state. When ignored, the persisted state is used.